docs/users_guide/8.12.1-notes.rst

```diff
@@ -10,6 +10,17 @@ following sections.
 Highlights
 ----------
 
+* The :extension:`LinearTypes` extension enables linear function syntax
+  ``a #-> b``, as described in `Linear Haskell: practical linearity in
+  a higher-order polymorphic language
+  <https://www.microsoft.com/en-us/research/publication/linear-haskell-practical-linearity-higher-order-polymorphic-language/>`__.
```

Comment:
We should link the proposal, rather than the paper, I think. https://github.com/ghc-proposals/ghc-proposals/blob/master/proposals/0111-linear-types.rst